packages/eslint-plugin/src/tests/rules/mui-require-container-property.test.ts

```diff
@@ -153,6 +168,18 @@ ruleTester.run<string, readonly unknown[]>(
           },
         ],
       },
+      {
+        code: `import { Select } from '@mui/material';
+               const props = {};
+               <Select {...props} />`,
```

Comment:
Why should we extend test cases with `<Select {...props} />`? Maybe I'm wrong, isn't it `useMenuButton(...)` used only for the `<Menu {...menuProps} />`?

Reply:
You're correct, i'm just testing edge cases on pipelines

Reply:
Ok, man, I won't review the draft. I'm just excited.